README.md

```
@@ -48,10 +48,97 @@ export function ExampleScreen() {
+
+<AppModal 
+  name="confirmation-dialog" 
+  visible={isVisible} 
+  priority={10}
+  onRequestClose={() => setIsVisible(false)}
+>
+  <YourModalContent />
+</AppModal>
+```
+
+2. **Set priorities wisely**: Higher numbers have precedence. Common priority patterns:
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)